star-ros2/src/star_safety_monitor/test/test_safety_monitor.cpp

```
@@ -19,50 +19,44 @@
 constexpr auto SPIN_TIMEOUT = std::chrono::milliseconds(100);
 constexpr double MESSAGE_WAIT_TIMEOUT_S = 2.0;
-}
+} // namespace
 
-class SafetyMonitorTest : public ::testing::Test
-{
+class SafetyMonitorTest : public ::testing::Test {
 protected:
-  void SetUp() override
-  {
+  void SetUp() override {
```

Clang (14.0.6)

[warning] method 'SetUp' can be made static

(readability-convert-member-functions-to-static)